source/blog/2016-03-04-atomic-site-on-atomic-app.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Projectatomic.io Running in Atomic App
+author: jberkus
+date: 2016-03-04 12:00:00 UTC
+tags: atomic.app, Nulecule, testing, development
+published: true
+comments: true
+---
+
+Since Atomic.app is getting close to its 1.0 release, I decided it was past time to make the atomic-site into an atomic.app instead of using a shell script that wraps Docker to test it. The new setup is a big improvement, and a useful guide to "Nuleculeizing" your own apps.
```

Comment:
Also.. should we allude to 1.0 release considering our discussions over the past few days? Maybe choosing to leave that part out would be wise.